lib/src/widgets/movie_detail.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import '../../utils/constants.dart';
+import 'movie_detail_button.dart';
+import 'movie_detail_image.dart';
+import 'movie_detail_information.dart';
+import 'movie_detail_overview.dart';
+import 'movie_detail_title.dart';
+
+class MovieDetail extends StatelessWidget {
+  final String movieTitle;
+  final String moviePosterPath;
+  final String movieReleaseDate;
+  final double movieVoteAverage;
```

Comment:
You can change this to String to avoid the conversion inside this widget and when you call it you can transform it inside the constructor. 
For example: 
MovieDetail(
         .....
         movieVoteAverage: result[index].voteAverage.toString() 
...)

Reply:
fixed